From 29c3c3d3c1b0c3349c99de4816bb08118700ae31 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Sun, 22 Jan 2012 16:17:27 -0500
Subject: [PATCH 353/933] Remove #warning

Keep a comment about future work, but don't generate compiler warnings.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
---
 drivers/staging/lttng/wrapper/poll.h        | 5 ++++-
 drivers/staging/lttng/wrapper/trace-clock.h | 8 ++++++--
 2 files changed, 10 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/lttng/wrapper/poll.h b/drivers/staging/lttng/wrapper/poll.h
index 080717b..540be31 100644
--- a/drivers/staging/lttng/wrapper/poll.h
+++ b/drivers/staging/lttng/wrapper/poll.h
@@ -9,7 +9,10 @@
 
 #include <linux/poll.h>
 
-#warning "poll_wait_set_exclusive() is defined as no-op. Thundering herd effect can be noticed with large number of consumer threads."
+/*
+ * Note: poll_wait_set_exclusive() is defined as no-op. Thundering herd
+ * effect can be noticed with large number of consumer threads.
+ */
 
 #define poll_wait_set_exclusive(poll_table)
 
diff --git a/drivers/staging/lttng/wrapper/trace-clock.h b/drivers/staging/lttng/wrapper/trace-clock.h
index 05ab055..d257972 100644
--- a/drivers/staging/lttng/wrapper/trace-clock.h
+++ b/drivers/staging/lttng/wrapper/trace-clock.h
@@ -56,6 +56,12 @@ static inline u32 trace_clock_freq_scale(void)
 
 static inline int get_trace_clock(void)
 {
+	/*
+	 * LTTng: Using mainline kernel monotonic clock. NMIs will not be
+	 * traced, and expect significant performance degradation compared to
+	 * the LTTng trace clocks. Integration of the LTTng 0.x trace clocks
+	 * into LTTng 2.0 is planned in a near future.
+	 */
 	printk(KERN_WARNING "LTTng: Using mainline kernel monotonic clock.\n");
 	printk(KERN_WARNING "  * NMIs will not be traced,\n");
 	printk(KERN_WARNING "  * expect significant performance degradation compared to the\n");
@@ -63,8 +69,6 @@ static inline int get_trace_clock(void)
 	printk(KERN_WARNING "Integration of the LTTng 0.x trace clocks into LTTng 2.0 is planned\n");
 	printk(KERN_WARNING "in a near future.\n");
 
-#warning "LTTng: Using mainline kernel monotonic clock. NMIs will not be traced, and expect significant performance degradation compared to the LTTng trace clocks. Integration of the LTTng 0.x trace clocks into LTTng 2.0 is planned in a near future."
-
 	return 0;
 }
 
-- 
2.0.2

